.github/workflows/build_and_test.yml

```diff
@@ -428,3 +428,35 @@ jobs:
     - name: Build with SBT
       run: |
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Phadoop-2.7 compile test:compile
+
+  tpcds-1g:
+    name: Run TPC-DS queries with SF=1
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+    - name: Checkout TPC-DS (SF=1) generated data repository
+      uses: actions/checkout@v2
+      with:
+        repository: maropu/spark-tpcds-sf-1
+        path: ./tpcds-sf-1
+        lfs: true
```

Comment:
Some files exceed the maximum single file size (100mb) defined by GitHub, so  I used `lfs` to upload them.

Reply:
To reduce the network traffic, shall we use more storage-efficient format like gzip or zstd? For example, the following is `zstd` result.
```
$ aws s3 ls s3://ttl-35d-813987666268/users/dongjoon/tpcds/1g-zstd/ --recursive --human-readable | grep MiB | sort -k3
2021-03-12 14:06:32    1.0 MiB users/dongjoon/tpcds/1g-zstd/item/part-00000-cc566e52-0113-4bb3-9fe5-ba69de177fc2-c000.zstd.parquet
2021-03-12 14:04:43    3.6 MiB users/dongjoon/tpcds/1g-zstd/customer/part-00000-068120d1-27e5-4553-8d2d-e75143aa1da4-c000.zstd.parquet
2021-03-12 14:03:49    4.2 MiB users/dongjoon/tpcds/1g-zstd/web_returns/part-00000-f5ef2333-1d55-4655-9e4f-55072eb88e41-c000.zstd.parquet
2021-03-12 14:05:19    4.9 MiB users/dongjoon/tpcds/1g-zstd/customer_demographics/part-00000-279d7119-dc0f-4267-a190-977a630c5443-c000.zstd.parquet
2021-03-12 14:00:48    7.8 MiB users/dongjoon/tpcds/1g-zstd/catalog_returns/part-00000-eefacec3-ac03-4202-bbaf-7fc9d5caa11a-c000.zstd.parquet
2021-03-12 14:02:51   12.1 MiB users/dongjoon/tpcds/1g-zstd/store_returns/part-00000-bb811271-b6c3-4e33-be26-4208c7d6fe7a-c000.zstd.parquet
2021-03-12 14:01:13   14.9 MiB users/dongjoon/tpcds/1g-zstd/inventory/part-00000-27cbda33-87d3-4f2a-8df4-2c2fd63b420c-c000.zstd.parquet
2021-03-12 14:03:19   34.4 MiB users/dongjoon/tpcds/1g-zstd/web_sales/part-00000-b9dff830-93f2-4514-ab2f-4cb3f4c7a704-c000.zstd.parquet
2021-03-12 14:00:08   72.6 MiB users/dongjoon/tpcds/1g-zstd/catalog_sales/part-00000-7b3d90bc-1bd3-4396-a039-c11b2fad45fa-c000.zstd.parquet
2021-03-12 14:01:54   91.4 MiB users/dongjoon/tpcds/1g-zstd/store_sales/part-00000-553b44ad-8032-4b0f-a34b-f72c41d17787-c000.zstd.parquet
```

Reply:
Since we are still not using Apache Parquet 1.12.0, `gzip` will be the only available option here.

Reply:
Ah, I see. Nice suggestion. I'll follow it.

Reply:
Another option is we can generate data at runtime: https://github.com/apache/spark/pull/31303

Reply:
okay, I've changed the format from snappy to gzip.

Reply:
> Another option is we can generate data at runtime: #31303

Yea, we can, but I think we don't need to re-generate these data per each invoke.